l5kit/l5kit/simulation/utils.py

```diff
@@ -74,3 +74,50 @@ def disable_agents(dataset: ChunkedDataset, allowlist: np.ndarray) -> None:
     dataset.agents["yaw"][mask_disable] *= 0
     dataset.agents["extent"][mask_disable] *= 0
     dataset.agents["label_probabilities"][mask_disable] = -1
+
+
+def get_frames_subset(dataset: ChunkedDataset, frame_start_idx: int, frame_end_idx: int) -> ChunkedDataset:
+    """Get a new dataset with frames between start (included) and end (excluded).
+    Assumptions:
+        the dataset has only 1 scene
+        the dataset is in numpy format and not zarr anymore
```

Comment:
I think this will be rendered all together in Sphinx (not sure though)

Reply:
because of the missing return?

Reply:
Because of the line break, I think you need a `-` in front of it or another line-break.